tern/command_lib/base.yml

```diff
@@ -94,7 +94,7 @@ dpkg:
       1:
         container:
           - "pkgs=`dpkg --get-selections | cut -f1 -d':' | awk '{print $1}'`"
-          - "for p in $pkgs; do cat /usr/share/doc/$p/copyright; echo LICF; done"
+          - "for p in $pkgs; do /bin/cat /usr/share/doc/$p/copyright; echo LICF; done"
```

Comment:
Is this change more related to resolving #565 ?

Reply:
If so, please move to its own PR :)

Reply:
yes, it will address issue #565 as well. Here, I have submitted this change because I am using `debut` external utility to parse Debian copyright data which accepts copyright file location instead of copyright data.

Reply:
Not sure what @nishakm thinks but I personally think that if it's addressing another bug it should be it's own PR or at least a separate commit in this PR. Your commit message says "It proposed a change in base.yml file to extract copyright file location instead of copyright text." but you also say that this is addressing but #565. The change in this PR only seems to be changing/specifying the location of the cat utility (which addresses #565) -- it's not clear to me how this changes the information that is retrieved as your commit message suggests.

Reply:
@rnjudge I can understand your point is but the two changes which I proposed through this PR are related to each other. If I don't make a change in `base.yml`, I can't parse copyright text of any Debian package using `debut` utility. It is a pure coincidence that another issue is already being notified which requires the same change of `base.yml`. Although, I am ok to dedicate this `base.yml` change to issue #565. Anyway, the change in `base.yml` is the base of parsing copyright text.

Reply:
If you want to keep the commits together that's fine but can you update your commit message to more accurately describe the changes you're making? The change to base.yml doesn't `extract copyright file location instead of copyright text` itself -- it may enable debut to do this -- but the change in base.yml as far as Tern code base is concerned is only specifying the location of the cat utility. Please also mention that this simultaneously works towards issue #565 somewhere in your commit message. Thank you!

Reply:
Sure!

Reply:
Thank you! Really appreciate this PR!

Reply:
@rnjudge, Done! I have moved the `base.yml` to a separate pull request.